tests/pytests/test_debug_commands.py

```diff
@@ -295,3 +289,13 @@ def testDumpHNSW(env):
     env.expect('FT.DEBUG', 'DUMP_HNSW', 'temp-idx', 'v_HNSW').\
         equal([['Doc id', 1, ['Neighbors in level 0', 2]], ['Doc id', 2, ['Neighbors in level 0', 1]],
                "Doc id 3 doesn't contain the given field"])
+
+@skip(cluster=False)
+def testCoordDebug(env: Env):
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 'name', 'TEXT').ok()
+    # Sanity check - regular debug command
+    env.expect(debug_cmd(), 'DUMP_TERMS', 'idx').equal([])
+    # Test Coordinator only debug command
+    env.expect(debug_cmd(), 'SHARD_CONNECTION_STATES').noError()
```

Comment:
Should `FT.DEBUG SHARD_CONNECTION_STATES` call fail somehow? Is there a point in adding a sanity test for the configuration return value itself?

Reply:
Only if we fails to register it. I just wanted to verify it is recognized